spec/features/favorites/favorite_delete_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe "After a user adds a favorite pet" do
+  it "displays a remove pet button instead of add favorite pet button" do
+
+    pet1 = create(:pet)
+
+    visit "/pets/#{pet1.id}" 
+
+    within("#pet-#{pet1.id}") do
```

Comment:
Curious to know why you used "within" here, I don't quite understand the functionality of that yet. Debrief at next stand up?